ts/parts/Tooltip.ts

```diff
@@ -1385,6 +1380,31 @@ H.Tooltip.prototype = {
                     yAxis.pos + Math.max(0, Math.min(point.plotY, yAxis.len))
             });
         });
+
+        /* If we have a seperate tooltip container, then update the necessary
+         * container properties.
+         * Test that tooltip has its own container and renderer before executing
+         * the operation.
+         */
+        const {
+            container,
+            outside,
+            renderer
+        } = tooltip;
+        if (outside && container && renderer) {
+            // Position the tooltip container to the chart container
+            const offset = H.offset(chart.container);
```

Comment:
Change this to use the existing `chart.pointer.chartPosition`, or get it if undefined. Avoids traversion the DOM unnecessarily. Assigned to @TorsteinHonsi .